.changeset/cool-months-tickle.md

```diff
@@ -0,0 +1,17 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab': minor
```

Comment:
Can be a patch release as the plugin is `<1.0.0`

Reply:
Updated